backend/app/tests/integration/conftest.py

```diff
@@ -1,6 +1,66 @@
 import pytest
 
 
-@pytest.fixture(scope="module")
-def api_url():
-    return "http://backend/api"
+import os
+
+import pytest
+
+
+# Get values we need to setup the environment from the command line or through env vars
+def pytest_addoption(parser):
+    group = parser.getgroup("corgi", "corgi")
+    group.addoption(
+        "--stack-name",
+        default=os.getenv("STACK_NAME", "dev"),
+        help="The name of the stack, used in the version endpoint",
+    )
+    parser.addoption(
+        "--tag",
+        default=os.getenv("TAG", "dev"),
+        help="The tag used on the images",
+    )
+    parser.addoption(
+        "--revision",
+        default=os.getenv("REVISION", "dev")
+    )
+
+    parser.addoption(
```

Comment:
We don't pass in DEPLOYED_AT anymore. We validate for a valid datetime string we can parse. I need to remove this option.

Reply:
This is resolved.

Reply:
This is resolved.